packages/design-system/src/components/Button/Button.spec.tsx

```diff
@@ -1,34 +1,37 @@
-/// <reference types="cypress" />
+/// <reference path="../../../cypress/support/index.d.ts" />
```

Comment:
```suggestion
/// <reference path="cypress/support/index.d.ts" />
```
Wouldn't that work with the `rootDirt` setup we have because of `~docs`?